buildstockbatch/gcp/gcp.py

```diff
@@ -586,10 +587,40 @@ def run_batch(self):
 
         # Start the job!
         created_job = client.create_job(create_request)
+        job_name = created_job.name
+
+        logger.info("Newly created GCP Batch job")
+        logger.info(f"  Job name: {job_name}")
+        logger.info(f"  Job UID: {created_job.uid}")
+        job_url = (
+            "https://console.cloud.google.com/batch/jobsDetail/regions/"
+            f"{self.region}/jobs/{self.unique_job_id}/details?project={self.gcp_project}"
+        )
+        logger.info(f"View GCP Batch job at {job_url}")
 
-        logger.info('Newly created GCP Batch job')
-        logger.info(f'  Job name: {created_job.name}')
-        logger.info(f'  Job UID: {created_job.uid}')
+        # Monitor job status while waiting for the job to complete
+        n_succeeded_last_time = 0
+        client = batch_v1.BatchServiceClient()
+        with tqdm.tqdm(desc="Running Simulations", total=task_count, unit="batch") as progress_bar:
+            job_status = None
+            while job_status not in ("SUCCEEDED", "FAILED", "DELETION_IN_PROGRESS"):
+                time.sleep(10)
+                job_info = client.get_job(batch_v1.GetJobRequest(name=job_name))
+                job_status = job_info.status.state.name
```

Comment:
Does it make sense to continue past this if job_status is not "SUCCEEDED"?

Reply:
Yes. If the state is QUEUED/SCHEDULED/RUNNING, then obviously we need to keep going. If the state is FAILED, then all the tasks will have finished, so this will just record how many of them failed or succeeded before stopping. If it's DELETION_IN_PROGRESS, it doesn't matter that much what we do, but I think checking the task counts one last time is fine.